python/lsst/verify/bin/inspectjob.py

```diff
@@ -121,7 +121,10 @@ def inspect_job(job):
 
     print("\nMeasurements:")
     for metric, measurement in job.measurements.items():
-        pretty_quantity = measurement.quantity.round(4)
+        if (measurement.quantity.value < 1e-2):
+            pretty_quantity = '{0.value:0.2e} {0.unit}'.format(measurement.quantity)
+        else:
+            pretty_quantity = '{0.value:0.2f} {0.unit}'.format(measurement.quantity)
```

Comment:
Can you use `0.4g` (or maybe `0.6g`, to allow for digits before the decimal point) here instead of having an explicit branch?

Reply:
I think the explicit branch is needed in case more than 4 sig. figures are present.